notebooks/additional-methods.ipynb

```diff
@@ -0,0 +1,589 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "curve = tonyear.get_baseline_curve(\"joos_2013\")\n",
+    "delays = np.arange(1, 101)\n",
+    "time_horizon = 100\n",
+    "discount_rate = 0\n",
+    "\n",
+    "equivalence_lashof = [\n",
+    "    1\n",
+    "    / tonyear.calculate_tonyears(\"lashof\", curve, time_horizon, delay, discount_rate)[\n",
+    "        \"num_for_equivalence\"\n",
+    "    ]\n",
+    "    for delay in delays\n",
+    "]\n",
+    "equivalence_car = [\n",
+    "    1 / calculate_tonyears(\"car\", curve, time_horizon, delay, discount_rate)[\"num_for_equivalence\"]\n",
+    "    for delay in delays\n",
+    "]\n",
+    "equivalence_qc = [\n",
+    "    1 / calculate_tonyears(\"qc\", curve, time_horizon, delay, discount_rate)[\"num_for_equivalence\"]\n",
+    "    for delay in delays\n",
+    "]\n",
+    "\n",
+    "plt.plot(np.array(equivalence_qc) / np.array(equivalence_lashof), label=\"Quebec/Lashof\")\n",
+    "plt.plot(np.array(equivalence_car) / np.array(equivalence_lashof), label=\"CAR/Lashof\")\n",
+    "\n",
+    "plt.xlabel(\"Delay\")\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f3a4a15",
+   "metadata": {},
+   "source": [
+    "While all three methods converge on 100 years of carbon storage being equivalent to a 1 tCO2\n",
+    "emission, they diverge for shorter storage periods. The Quebecoi method, in particular, overvalues\n",
```

Comment:
Is Quebecoi the correct term? Or does it refer to _people_ from Quebec? I'm not an expert here, but if Idaho developed a method, it would be the `Idaho Method`, not the `Idahoan Method`

Reply:
Great call. Getting rid of quebecois everywhere.